### DISCUSSION TITLE
Where can I find a proper documentation for all types of parameters and return values?

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [vfsfitvnm/frida-il2cpp-bridge#456] [Q&A]
Like

`{"type":"error","description":"\u001b[0m\u001b[38;5;9mil2cpp\u001b[0m: \u001b[0mcannot invoke non-static method SetGameOver as it must be invoked throught a Il2Cpp.Object, not a Il2Cpp.Class","stack":"\u001b[0m\u001b[38;5;9mil2cpp\u001b[0m: \u001b[0mcannot invoke non-static method SetGameOver as it must be invoked throught a Il2Cpp.Object, not a Il2Cpp.Class\u001b[3m\u001b[2m\n    at invoke (il2cpp/structs/method.ts:215)\n    at <anonymous> (index.ts:13)\n    at call (native)\n    at <anonymous> (il2cpp/structs/method.ts:354)\u001b[0m","fileName":"utils/console.ts","lineNumber":3,"columnNumber":1}`

I'm getting this error while returning a boolean value. I know I'm doing it wrong and foolish, but I did many tries, but I can't get to achieve it

`import "frida-il2cpp-bridge";

Il2Cpp.perform(function(){
    const assemblyc = Il2Cpp.domain.assembly("Assembly-CSharp").image;
    const gamecontroller = assemblyc.class("GameController");
    const setgameover = gamecontroller.method("SetGameOver");
    
    setgameover.implementation = function (revive) {
        console.log(revive);
        setgameover.invoke(true);
    };
});`

Original class
`public Void SetGameOver(Boolean allowRevive) { }`

**Comment by RevealedSoulEven** [comment]
![Screenshot_20231230-075739](https://github.com/vfsfitvnm/frida-il2cpp-bridge/assets/49313374/ba809e76-f8bd-4fcb-97d5-9d3105cf4c70)


![Screenshot_20231230-075722](https://github.com/vfsfitvnm/frida-il2cpp-bridge/assets/49313374/ad5aaf9e-a8f2-4157-99a3-6b9731c153fa)
I couldn't figure out, what I'm doing wrong. Please help😶‍🌫️

**Reply by vfsfitvnm** [reply]
add
```ts
// @ts-ignore
```
on line 11

**Reply by RevealedSoulEven** [reply]
I fixed that anyway, I didn't used boolean and object, and it worked somehow
